lib/utils/getTransactionOptions/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { urls } from "../../constants";
+
+/** Returns the fast gas price from polygonscan plus 10 percent*/
+export const getGasPrice = async () => {
+  try {
+    const response = await fetch(urls.polyscanGasTracker);
+    const data = await response.json();
+    if (data.message == "OK") {
+      const targetGasPrice = (
+        parseFloat(data.result.FastGasPrice) * 1.1
+      ).toFixed(2);
+      return ethers.utils.parseUnits(targetGasPrice.toString(), "gwei");
```

Comment:
I'd like to move away from the names and use the decimals count instead
```suggestion
      return ethers.utils.parseUnits(targetGasPrice.toString(), 18);
```